jaxws-ri/tests/pom.xml

```diff
@@ -64,8 +63,14 @@
                     <transport>${ws.transport}</transport>
                     <extraVmArgs>${ws.jvmOpts}</extraVmArgs>
                     <vmArgs>
-                        <!-- ensure that you are specifying PermGen jvm options only for jdk < 8 -->
-                        <vmArg>-Xms256m -Xmx512m ${ws.permGenJvmOpts}</vmArg>
+                        <vmArg>-Xms256m -Xmx512m</vmArg>
+                        <vmArg>--add-opens java.base/java.lang=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.base/jdk.internal.misc=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.base/java.util=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.xml/com.sun.org.apache.xalan.internal.xsltc.trax=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.xml/com.sun.org.apache.xerces.internal.jaxp=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.xml/com.sun.org.apache.xerces.internal.dom=ALL-UNNAMED</vmArg>
+                        <vmArg>--add-opens java.xml/com.sun.org.apache.xerces.internal.jaxp.datatype=ALL-UNNAMED</vmArg>
```

Comment:
this should not be needed on JDK11+, since xmlresolver which requires this to be set, should not be used. [ws-test-harness](https://github.com/eclipse-ee4j/metro-ws-test-harness) may need to be updated instead.

Reply:
This is the stack trace when that is not added:

```
2) fromwsdl.handler_simple.client testDispatchSourceSOAPHandler(com.sun.xml.ws.test.exec.ClientExecutor)Sourced file: testDispatchSourceSOAPHandler unknown error: Unable to make public javax.xml.transform.Transformer com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl.newTransformer() throws javax.xml.transform.TransformerConfigurationException accessible: module java.xml does not "exports com.sun.org.apache.xalan.internal.xsltc.trax" to unnamed module @7daf6ecc : at Line: 26 : in file: testDispatchSourceSOAPHandler : int responseInt = getIntFromResponse ( response ) 

Called from method: getIntFromResponse : at Line: 26 : in file: testDispatchSourceSOAPHandler : getIntFromResponse ( response ) 
	at bsh.Interpreter.eval(Interpreter.java:706)
	at com.sun.xml.ws.test.client.InterpreterEx.eval(InterpreterEx.java:60)
	at com.sun.xml.ws.test.exec.ClientExecutor.invoke(ClientExecutor.java:141)
	at com.sun.xml.ws.test.exec.ClientExecutor.runTest(ClientExecutor.java:110)
	at com.sun.xml.ws.test.Main$1.doRun(Main.java:325)
	at com.sun.xml.ws.test.Main.run(Main.java:334)
	at com.sun.xml.ws.test.Main.doMain(Main.java:254)
	at com.sun.xml.ws.test.Main.main(Main.java:239)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make public javax.xml.transform.Transformer com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl.newTransformer() throws javax.xml.transform.TransformerConfigurationException accessible: module java.xml does not "exports com.sun.org.apache.xalan.internal.xsltc.trax" to unnamed module @7daf6ecc
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
	at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
	at bsh.reflect.ReflectManagerImpl.setAccessible(ReflectManagerImpl.java:53)
	at bsh.ReflectManager.RMSetAccessible(ReflectManager.java:72)
	at bsh.Reflect.resolveJavaMethod(Reflect.java:454)
	at bsh.Reflect.resolveExpectedJavaMethod(Reflect.java:382)
	at bsh.Reflect.invokeObjectMethod(Reflect.java:74)
	at bsh.BSHPrimarySuffix.doName(BSHPrimarySuffix.java:170)
	at bsh.BSHPrimarySuffix.doSuffix(BSHPrimarySuffix.java:114)
	at bsh.BSHPrimaryExpression.eval(BSHPrimaryExpression.java:74)
	at bsh.BSHPrimaryExpression.eval(BSHPrimaryExpression.java:41)
	at bsh.BSHVariableDeclarator.eval(BSHVariableDeclarator.java:80)
	at bsh.BSHTypedVariableDeclaration.eval(BSHTypedVariableDeclaration.java:78)
	at bsh.BSHBlock.evalBlock(BSHBlock.java:125)
	at bsh.BSHBlock.eval(BSHBlock.java:75)
	at bsh.BshMethod.invokeImpl(BshMethod.java:356)
	at bsh.BshMethod.invoke(BshMethod.java:252)
	at bsh.BshMethod.invoke(BshMethod.java:180)
	at bsh.Name.invokeLocalMethod(Name.java:911)
	at bsh.Name.invokeMethod(Name.java:798)
	at bsh.BSHMethodInvocation.eval(BSHMethodInvocation.java:69)
	at bsh.BSHPrimaryExpression.eval(BSHPrimaryExpression.java:96)
	at bsh.BSHPrimaryExpression.eval(BSHPrimaryExpression.java:41)
	at bsh.BSHVariableDeclarator.eval(BSHVariableDeclarator.java:80)
	at bsh.BSHTypedVariableDeclaration.eval(BSHTypedVariableDeclaration.java:78)
	at bsh.Interpreter.eval(Interpreter.java:659)

```
It looks the issue is in the metro-test-harness project with this dependency:
```
            <dependency>
                <groupId>org.apache-extras.beanshell</groupId>
                <artifactId>bsh</artifactId>
                <version>2.0b6</version>
            </dependency>
```
But I don't see a newer version in maven to update it.